src/components/widgets/ColorWidget.js

```diff
@@ -1,22 +1,30 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-function ColorWidget(props) {
-  const { disabled, readonly, registry: { widgets: { BaseInput } } } = props;
-  return <BaseInput type="color" {...props} disabled={disabled || readonly} />;
-}
+import { ColorPicker } from '@carecloud/material-cuil';
 
-if (process.env.NODE_ENV !== 'production') {
-  ColorWidget.propTypes = {
-    schema: PropTypes.object.isRequired,
-    id: PropTypes.string.isRequired,
-    value: PropTypes.string,
-    required: PropTypes.bool,
-    disabled: PropTypes.bool,
-    readonly: PropTypes.bool,
-    autofocus: PropTypes.bool,
-    onChange: PropTypes.func,
-  };
-}
+const ColorWidget = props => {
+  const {
+    value,
+    /**
+     * @description By default, `react-jsonschema-form` is going to disable
+     * the alpha value in the picker
+     * @default `true`
+     * @type {Boolean}
+     */
+    disableAlpha = true,
+    ...otherProps
+  } = props;
+
+  return <ColorPicker color={value} disableAlpha={disableAlpha} {...otherProps} />;
```

Comment:
The disabled property should be passed as `disabled={disabled || readonly}`.

This component will receive `schema` and `autofocus` properties, that `ColorPicker` does not understand, so they should be not passed down to it.

Reply:
Done!